src/cromshell/submit/command.py

```diff
@@ -105,8 +105,9 @@ def main(config, wdl, wdl_json, options_json, dependencies_zip):
     # TODO: Refactor these file manipulations into its own "cleanup" function?
     # If we get here, we successfully submitted the job and should track it locally:
     # 1. Create a directory to hold function input files, using server name
+    server_folder_name = config.get_local_folder_name()
```

Comment:
Instead of using a variable that was set at the initialization of cromshellconfig.py which would have been set to the default cromshell server, we now use a function to obtain the server name so that if the user-specified their server we would get their specified server instead of the default.